saleor/warehouse/management.py

```diff
@@ -198,12 +211,20 @@ def increase_stock(
     function increase `quantity_allocated`. If allocation does not exist function
     create a new allocation for this order line in this stock.
     """
+    plugins_manager = get_plugins_manager()
+    product_variant_stock_trigger = ProductVariantStockWebhookTrigger()
     stock = (
         Stock.objects.select_for_update(of=("self",))
         .filter(warehouse=warehouse, product_variant=order_line.variant)
         .first()
     )
     if stock:
+        product_variant_stock_trigger.append_stock_data(
+            stock, False, stock.quantity, quantity
+        )
+        product_variant_stock_trigger.trigger_product_variant_back_in_stock_webhook(
+            plugins_manager
+        )
         stock.increase_stock(quantity, commit=True)
     else:
         stock = Stock.objects.create(
```

Comment:
Also 👇  ( I can't comment below lines but this comment is to 233-242.) there is a possibility that stock will be immediately allocated which means that back_in_stock is not true.